src/utils.ts

```diff
@@ -80,3 +80,8 @@ export const pickOne = (arr: any[]) =>
  * Boolean generator
  */
 export const dice = () => !!Math.round(Math.random());
+
+/**
+ * Polyfill for Array.prototype.flat
+ */
+export const flat = (arr: any[][]) => arr.reduce((acc, val) => acc.concat(val), []);
```

Comment:
@walmik I personally don't think this function belongs in the project. Do you know what needs to be done for this function to be properly available without the polyfill?